db/migrate/20230404210801_create_comment.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComment < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.references :user, null: false, foreign_key: true
```

Comment:
- [ ] Kindly edit this reference so the foreign key supposes to be `author_id` but what you did here will make Active Record create it as `user_id` because of naming conventions.
[hint]: make this reference the same one you created at the posts table migration:
```sql
t.references :author, null: false, foreign_key: {to_table: :users}
```

Reply:
<img width="500" alt="Captura de pantalla 2023-04-04 a la(s) 16 48 52" src="https://user-images.githubusercontent.com/99061965/229940706-5d88c577-41ff-4934-bbe5-a89c494c6cbc.png">